src/main/java/ma/boot/springboot/service/read/ReadingCsvFile.java

```diff
@@ -0,0 +1,66 @@
+package ma.boot.springboot.service.read;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import ma.boot.springboot.model.Review;
+import org.apache.log4j.Logger;
+import org.supercsv.cellprocessor.CellProcessorAdaptor;
+import org.supercsv.cellprocessor.ParseInt;
+import org.supercsv.cellprocessor.ParseLong;
+import org.supercsv.cellprocessor.constraint.NotNull;
+import org.supercsv.cellprocessor.ift.CellProcessor;
+import org.supercsv.exception.SuperCsvException;
+import org.supercsv.io.CsvBeanReader;
+import org.supercsv.io.ICsvBeanReader;
+import org.supercsv.prefs.CsvPreference;
+import org.supercsv.util.CsvContext;
+
+public class ReadingCsvFile {
+    private static final Logger logger = Logger.getLogger(ReadingCsvFile.class);
+
+    public static int readWithCsvBeanReader(String fileName) throws IOException {
```

Comment:
this method should return list of reviews. 
And if you need number of records just call reviews.size()

Reply:
Realised this in SB-2 and changed it there. Will do it here as well